Cataclysm/Rogue.lua

```diff
@@ -1331,7 +1429,7 @@ spec:RegisterAbilities( {
         texture = 135428,
 
         handler = function ()
-            -- TODO: Apply offhand poison.
+            dispelEnrageEffectsOnTarget()
```

Comment:
This function doesn't/shouldn't exist.  You can `removeDebuff( "target", "dispellable_enrage" )` if I'm remembering correctly, though.